bot/src/discord_commands/help.js

```diff
@@ -27,6 +27,7 @@ const COMMANDS_TO_GENERATE_HELP_FOR = [
   'anime',
   'shiritoriforever',
   'random',
+  'schedule',
```

Comment:
Until the schedules persist across reboots, we should omit this from the help so it's a hidden command (otherwise people are gonna be hollering at me about their schedules mysteriously stopping).